btree/btree.h

```diff
@@ -281,6 +283,28 @@ void m0_btree_iter(struct m0_btree *arbor, const struct m0_btree_key *key,
 		   const struct m0_btree_cb *cb, uint64_t flags,
 		   struct m0_btree_op *bop);
 
+/**
+ * Returns the records corresponding to minimum key of the btree.
+ *
+ * @param arbor Btree parameteres.`
+ * @param cb    Callback routine to return operation output.
+ * @param flags Operation specific flags (cookie, lockall etc.).
+ * @param bop   Btree operation related parameters.
+ */
+void m0_btree_minkey(struct m0_btree *arbor, const struct m0_btree_cb *cb,
+		     uint64_t flags, struct m0_btree_op *bop);
+
+/**
+ * Returns the records corresponding to maximum key of the btree.
+ *
+ * @param arbor Btree parameteres.`
```

Comment:
Extra apostrophe after full stop.
Kindly remove it.

Reply:
Done.